### PR TITLE
Delete classification icons

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a basic trash icon for classifications.

### CODE REVIEW

**Review by bfhealy — APPROVED** [review]
Looks good, this is a nice way to speed up classification deletion. I notice that the delete icon changes position slightly after I mouse over a classification. Is there a way to fix its location in the final position?

classification_trash_gif

**Comment by mcoughlin** [comment]
@bfhealy No idea.

**Comment by bfhealy** [comment]
@mcoughlin It shouldn't affect a scanner's ability to delete classifications quickly, at least.

File: static/js/components/ShowClassification.jsx [hunk 1]
```diff
@@ -1,22 +1,72 @@
...
 
   const classification = classifications[0];
+
+  const permission =
+    currentUser.permissions.includes("System admin") ||
+    currentUser.permissions.includes("Manage groups") ||
+    currentUser.username === classification.author_name;
+
+  console.log("classification", classification);
+  console.log("permission", permission);
```
Comment (guynir42) [inline static/js/components/ShowClassification.jsx:69]:
```suggestion
```

File: static/js/components/ShowClassification.jsx [hunk 2]
```diff
@@ -1,22 +1,72 @@
...
+        }
+      }
+    );
+  };
 
   const classification = classifications[0];
+
+  const permission =
+    currentUser.permissions.includes("System admin") ||
+    currentUser.permissions.includes("Manage groups") ||
```
Comment (guynir42) [inline static/js/components/ShowClassification.jsx:65]: Why "Manage groups"?

**Review by guynir42 — APPROVED** [review]
Seems like a nice solution.

File: static/js/components/ShowClassification.jsx (hunk 2 above)
Comment (mcoughlin) [reply static/js/components/ShowClassification.jsx:65]: @guynir42 That's just the permission we have on the classifications (I expect this is as close as we get to "person who manages source stuff so can manage classifications")

**Pull request merged**